drake/common/CMakeLists.txt

```diff
@@ -2,6 +2,12 @@
 # Drake common library
 #
 
+drake_find_package(Protobuf REQUIRED)
+include_directories(${Protobuf_INCLUDE_DIRS})
+include_directories(${PROJECT_BINARY_DIR}/..)  # to find generated protobuf headers
```

Comment:
@RussTedrake Check the last three commits I added to this branch: https://github.com/jamiesnape/drake/tree/call_matlab_protobuf